garak/generators/cohere.py

```diff
@@ -4,6 +4,9 @@
 but just supply the name of another either on the command line or as the
 constructor param if you want to use that. You'll need to set an environment
 variable called COHERE_API_KEY to your Cohere API key, for this generator.
+
+NOTE: This endpoint is end of support and only supports command model as of now
+https://docs.cohere.com/reference/generate
```

Comment:
Is it worth `raise`ing a DeprecationError when this model is instantiated with a name other than command?